src/jvm/main/org/jetbrains/kotlinx/lincheck/Utils.kt

```diff
@@ -25,6 +25,57 @@ import java.util.*
 import kotlin.coroutines.*
 import kotlin.coroutines.intrinsics.*
 
+fun shouldTransformClass(className: String): Boolean {
```

Comment:
It seems the code was moved from somewhere, but I do not see it removed in the original location.

Reply:
Also, I would suggest keeping the transformation-related logic in the corresponding package.